src/main/java/com/laytonsmith/abstraction/events/MCPlayerVelocityEvent.java

```diff
@@ -0,0 +1,19 @@
+package com.laytonsmith.abstraction.events;
+
+import com.laytonsmith.abstraction.MCPlayer;
+import com.laytonsmith.core.events.BindableEvent;
+import org.bukkit.util.Vector;
+
+public interface MCPlayerVelocityEvent extends BindableEvent {
+
+	Vector getVelocity();
+
+	MCPlayer getPlayer();
+
+	boolean isCancelled();
+
+	void setCancelled(boolean cancel);
+
+	void setVelocity(Vector velocity);
```

Comment:
Part 5: (continued from part 4) ... by having this method accept Vector3D... (see part 6)